src/main/java/org/verdictdb/coordinator/AsyncAggExecutionRulerByConverge.java

```diff
@@ -0,0 +1,126 @@
+package org.verdictdb.coordinator;
+
+import org.verdictdb.VerdictResultStream;
+import org.verdictdb.VerdictSingleResult;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AsyncAggExecutionRulerByConverge extends AsyncAggExecutionRuler {
+
+  VerdictResultStream resultStream;
+
+  Coordinator runningCoordinator;
+
+  // the values of the result should be within [(1-valueError), (1+valueError)] of the previous result.
```

Comment:
Clarify that the new value should be within [(1-valueError)*prevValue, (1+valueError)*prevValue].